rb_ws/src/buggy/scripts/auton/path_planner.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+
+import rospy
+from sensor_msgs.msg import NavSatFix
+from std_msgs.msg import Float64
+from pose import Pose
+
+from occupancy_grid.grid_manager import OccupancyGrid
+from path_projection import Projector
+from trajectory import Trajectory
+from world import World
+
+LOOKAHEAD_TIME = 2.0 #s
+RESOLUTION = 30 #samples/s
+PASSING_OFFSET = 2 #m
+# in meters, the number of meters behind NAND before we start morphing the trajectory
+REAR_MARGIN = 10
+
+# in meters, the number of meters in front of NAND,
+# before the morphed trajectory rejoins the nominal trajectory
+# WARNING: set this value to be greater than 10m/s * lookahead time (10 m/s is the upper limit
```

Comment:
Is it really only 10m/s? That's 22mph, I recall it being faster than that at some point but it's been a year.

Reply:
Checked last year's bags, and the top speed was just over 10.  Will amend this to 15m/s